README.md

```diff
@@ -53,6 +54,7 @@ Flags:
 |--username             |SLACK_USERNAME             |
 |--icon-url             |SLACK_ICON_URL             |
 |--description-template |SLACK_DESCRIPTION_TEMPLATE |
+|--alert-on-critical    |SENSU_SLACK_ALERT_CRITICAL |
```

Comment:
To be consistent with the naming of the other environment variables, it appears that this variable should be named "SLACK_ALERT_ON_CRITICAL".

Reply:
I have updated this in the latest push.  Thanks!